testing/test_bestest_air.py

```diff
@@ -43,6 +44,36 @@ def test_typical_cool_day(self):
     def test_mix_day(self):
         self.run_time_period('mix_day')
 
+    def test_scenario_flag(self):
+        '''Ensures the scenario flag is set properly.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        None
+
+        '''
+
+        length = 86400*14
+        # Get current step
+        step_current = requests.get('{0}/step'.format(self.url)).json()['payload']
+        # Initialize test case scenario
+        requests.put('{0}/scenario'.format(self.url), json={'time_period':'peak_heat_day'})
+        # Set simulation step
+        requests.put('{0}/step'.format(self.url), json={'step':length}).json()['payload']
+        # Simulation Loop
+        requests.post('{0}/advance'.format(self.url)).json()['payload']
```

Comment:
Same as above

Reply:
Same as above.

Reply:
Resolved in https://github.com/ibpsa/project1-boptest/pull/566/commits/fa5bfedb137bbe73cb0266814c121b2d0fb0b007.